### ISSUE TITLE
The problem of  BN quantization 

### ISSUE THREAD

**Issue by wmn931201** [openvinotoolkit/nncf#1560]
Hi,  

I am trying to running the nncf quantization with PyTorch.   Does NNCF quantify the weight and activation of BN or fuse BN to conv?

**Comment by vshampor** [comment]
Greetings!

NNCF in PyTorch performs quantization on weights of conv- and linear-like layers as they are present in the torch framework representation. BatchNorm layers are only quantized in terms of activations, and only if they are stand-alone, e.g. are not preceded by an operation that already profits from INT8 execution. Any BN optimizations such as BN to conv fusing are made at further levels of OpenVINO inference outside NNCF (e.g. in Model Optimizer).
